### ISSUE TITLE
Creating a CRD fails with "ValueError: Invalid value for `conditions`, must not be `None`"

### ISSUE THREAD

**Issue by ashpreetbedi** [kubernetes-client/python#1022]
**What happened (please include outputs or screenshots)**:

NOTE: This is a duplicate of https://github.com/kubernetes-client/python/issues/415 and https://github.com/kubernetes-client/python/issues/376 but those issues are closed.

I'm trying to create a CRD with python client 10.0.1 (`kubernetes==10.0.1`) and with Kubernetes server 1.14 using the `kubernetes.client.ApiextensionsV1beta1Api.create_custom_resource_definition() API`

Body for the CRD:
```
{
 'api_version': 'apiextensions.k8s.io/v1beta1',
 'kind': 'CustomResourceDefinition',
 'metadata': {
              'labels': {'app.kubernetes.io/component': 'ingressroutes.traefik.containo.us',
                         'app.kubernetes.io/part-of': 'traefik'},
              'name': 'ingressroutes.traefik.containo.us',
 },
 'spec': {
          'group': 'traefik.containo.us',
          'names': {
                    'kind': 'IngressRoute',
                    'plural': 'ingressroutes',
                    'singular': 'ingressroute'
          },
          'scope': 'Namespaced',
          'version': 'v1alpha1',
  },
}
```

This raises the following error:
```
    raise ValueError("Invalid value for `conditions`, must not be `None`")
ValueError: Invalid value for `conditions`, must not be `None`
```

This issue is also reported https://github.com/kubernetes-client/python/issues/415 and https://github.com/kubernetes-client/python/issues/376 but those issues are closed as stale

Could someone help me in finding a patch for this?

**What you expected to happen**:

CRD Should be created

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (`kubectl version`): 1.14
- OS (e.g., MacOS 10.13.6): MacOS 10.15
- Python version (`python --version`): 3.7.4
- Python client version (`pip list | grep kubernetes`): 10.0.1

**Comment by roycaihw** [comment]
This is fixed in upstream Kubernetes 1.16 https://github.com/kubernetes/kubernetes/pull/64996. The next major release of this client (v12.x) will include the fix

/assign

**Comment by ashpreetbedi** [comment]
Awesome, thanks @roycaihw

**Comment by acesir** [comment]
Is there currently a workaround for this?

**Comment by LouisPlisso** [comment]
When is version 12 planned to be released? 🥺 

We (MongoDB kubernetes operator) are currently blocked by this 😿
